### ISSUE TITLE
usage of python3 (generator-based) zip in _pixa_to_list causes segfault at pix.informat

### ISSUE THREAD

**Issue by glqstrauss** [sirfz/tesserocr#173]
OS:
`Mac OS X 10.14`
Python: 
`Python 3.6.6`
Tesserocr:
`tesserocr==2.4.0`
Tesseract:
```
tesseract 4.0.0
 leptonica-1.77.0
  libgif 5.1.4 : libjpeg 9c : libpng 1.6.36 : libtiff 4.0.10 : zlib 1.2.11 : libwebp 1.0.2 : libopenjp2 2.3.0
 Found AVX2
 Found AVX
 Found SSE
```

When running tesserocr unmodified, the following code produces a segfault at `_pix_to_image`'s invocation of `pix.informat`:
```
import locale
locale.setlocale(locale.LC_CTYPE, 'C')
import tesserocr
from PIL import Image
with tesserocr.PyTessBaseAPI() as api:
    im = Image.open('[LOCAL_PATH_TO_PROJECT_DIR]/tesserocr/tests/eurotext.tif')
    api.SetImage(im)
    api.Recognize()
    res = list(api.GetWords())
```

I found that by replacing
```
cdef pixa_to_list(Pixa *pixa):
    """Convert Pixa (Array of pixes and boxes) to list of pix, box tuples."""
    return zip((_pix_to_image(pix) for pix in pixa.pix[:pixa.n]), boxa_to_list(pixa.boxa))
```
with
```
cdef pixa_to_list(Pixa *pixa):
    """Convert Pixa (Array of pixes and boxes) to list of pix, box tuples."""
    tuples = []
    for pixbox_tuple in zip((_pix_to_image(pix) for pix in pixa.pix[:pixa.n]), boxa_to_list(pixa.boxa)):
        tuples.append(pixbox_tuple)
    return tuples
```
the segfault is avoided.

Assuming this has something to do with python3's generator-based implementation of `zip`, but wanted to make sure this is not some other issue with my particular versions of C/C++ libs.

**Comment by sirfz** [comment]
Good find, please submit a PR. I guess just wrapping `zip` with `list` is a "nicer" fix though.

**Comment by sirfz** [comment]
Fixed in PR #176
